tests/helpers.bash

```diff
@@ -483,7 +483,7 @@ function k8s_apply_policy {
     echo "  $i: ${currentRevison[$i]}"
   done
 
-  kubectl $action -f $policy
+  kubectl $action -f $policy 2>/dev/null
```

Comment:
What if kubectl fails for real? We won't see the error message. Can we fix the tests instead?